src/main/java/mate/academy/model/Movie.java

```diff
@@ -4,10 +4,8 @@
 import jakarta.persistence.GeneratedValue;
 import jakarta.persistence.GenerationType;
 import jakarta.persistence.Id;
-import jakarta.persistence.Table;
 
-@Entity
-@Table(name = "movies")
+@Entity(name = "movies")
```

Comment:
entity name and table name are not the same
you should read about it, or try creating entities, with defined entity name, and other entity with defined table name